javascript/javascript1/week4/index.js

```diff
@@ -0,0 +1,89 @@
+//JS week4
+const storeName = [];
+const listToDo =[];
+const numbers =[];
+
+function getReply(command){
+    if(command ==="Hello my name is Mariem" || command.includes("my name is Mariem")) {
+        let command1 = command.split(" ");
+        console.log (command1);
+        let userName = command1[command1.length-1];
+        storeName.push(userName);
+         console.log(`Nice to meet you ${userName}`)
+    }
+    if(command === "What is my name?" && storeName.length >= 1) {
+        console.log(`Your name is ${storeName[0]}`);
+    }else if (storeName.length == 0){
+        console.log('Your name is not here yet !');
+    }
+    if(command==="Add fishing to my todo"){
+        const listCommand =command.replace("Add fishing to my todo", "fishing")
+        listToDo.push(listCommand);
+        console.log("fishing added to your todo");
+        console.log(listToDo);
+    }
+    if(command==="Add singing in the shower to my todo"){
+        const listCommand2 =command.replace("Add singing in the shower to my todo", "singing in the shower")
+        listToDo.push(listCommand2);
+        console.log("Singing added to your todo");
+        console.log(listToDo);
+    }
+    for(let i=0; i<listToDo.length; i++){
```

Comment:
you need to remove if command is "Remove fishing from my todo"

Reply:
could you please  explain it more I can get it 😊😊